sycl/test-e2e/Basic/launch_queries/max_sub_group_size.cpp

```diff
@@ -0,0 +1,86 @@
+// REQUIRES aspect-subgroup
```

Comment:
There is no dedicated aspect for sub-groups ([SYCL 2020 spec](https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#sec:device-aspects)) and the same goes for the [launch queries extension spec](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/proposed/sycl_ext_oneapi_launch_queries.asciidoc) - it does not say that any of those queries require any aspects to be supported by a device, i.e. they should work on every device.

This test would have been always skipped if not for missing `:` which is required for a LIT directive to work.

The concept is required to be supported by every device, but device can say that it only provides a single sub-group per work-group, or say that maximum sub-group size is equal to 1 to indicate lack of support for the feature.